projects/core/src/services/database.service.ts

```diff
@@ -0,0 +1,42 @@
+import { HttpClient } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { AngularFireDatabase } from "@angular/fire/compat/database";
+import { Observable } from "rxjs";
+import { environment } from "src/environments/environment";
+
+export interface User {
+  uid: string;
+  email: string;
+  displayName: string;
+  photoURL: string;
+  emailVerified: boolean;
+}
+
+export interface Lesson {
+  name: string;
+  description: string;
+}
+
+export enum ListType {
+  users = "users",
+  lessons = "lessons",
+}
```

Comment:
it might be better to move the interfaces into a separate file

Reply:
It's just temporary interfaces related only for the first version.
But, make sense to move them out from database service, because database service should not know anythying about concrete entites.
Ex. users.data.service can contain it.

Reply:
Lessons will be presented in separated task, as well as User.

So, @iradamamedova you can move them into data services, but you also can leave them here.

Reply:
i have moved them into users and lessons services. is it ok if i leave listtype enum here?

Reply:
yes, BTW, name enum porperties in TITLE_CASE, ex. USERS